### PR TITLE
Pia 4098 drag to dismiss edit article modal screen UI layout issues

### PR DESCRIPTION
**Pull request by ValentinaIancu-Gini**

### CODE REVIEW

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/EditLineItem/EditLineItemView.swift [hunk 1]
```diff
@@ -59,61 +59,50 @@ final class EditLineItemView: UIView {
     private lazy var stackView: UIStackView = {
         let stackView = UIStackView()
         stackView.translatesAutoresizingMaskIntoConstraints = false
-        stackView.distribution = .fillEqually
+		stackView.distribution = .fill
```
Comment (ValentinaIancu-Gini) [inline BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/EditLineItem/EditLineItemView.swift:62]: Please ignore this indentation, it's github issue :(

File: BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/UIView+RoundCorners.swift [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+//  Created by Valentina Iancu on 30.06.23.
+//
+
+import UIKit
+
+extension UIView {
+	func round(corners: UIRectCorner = [UIRectCorner.allCorners], radius: CGFloat) {
+		layer.maskedCorners = CACornerMask(rawValue: corners.rawValue)
+		layer.cornerRadius = radius
+		layer.masksToBounds = radius != 0
```
Comment (zladzeyka) [inline BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/UIView+RoundCorners.swift:14]: Clarification: how should we read this?
		`layer.masksToBounds = radius != 0`
And why we don't need `!=0` for
		`layer.cornerRadius = radius`

**Review by zladzeyka — CHANGES_REQUESTED** [review]
Great job,
I've left a few tiny questions ☺️

**Comment by zladzeyka** [comment]
https://github.com/gini/gini-mobile-ios/assets/3911816/544fbd4b-162e-4d22-806e-8c171ab057c7

I've noticed one small layout issue if we rotate the screen and drag the presented screen up a bit, the screen will not update layout and going under the keyboard

**Review by zladzeyka — APPROVED** [review]

**Comment by zladzeyka** [comment]
> > RPReplay_Final1689088402.MP4
> > I've noticed one small layout issue if we rotate the screen and drag the presented screen up a bit, the screen will not update layout and going under the keyboard
> 
> Related to this, this is not something affected by these current changes :|. I've tested on the main branch and is also happening.
> 
>  Simulator.Screen.Recording.-.iPad.Air.2.-.2023-07-11.at.18.00.43.mp4 
> 
> Do you want me to fix it in my pr? Or should we create a separate ticket for this?

It will be a separate ticket

**Comment by ValentinaIancu-Gini** [comment]
> > > RPReplay_Final1689088402.MP4
> > > I've noticed one small layout issue if we rotate the screen and drag the presented screen up a bit, the screen will not update layout and going under the keyboard
> > 
> > 
> > Related to this, this is not something affected by these current changes :|. I've tested on the main branch and is also happening.
> > Simulator.Screen.Recording.-.iPad.Air.2.-.2023-07-11.at.18.00.43.mp4
> > Do you want me to fix it in my pr? Or should we create a separate ticket for this?
> 
> It will be a separate ticket

I will create a ticket for this!

**Pull request merged**